engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NkGlobalParameters.scala

```diff
@@ -36,7 +36,9 @@ case class NkGlobalParameters(buildInfo: String,
 //this is part of global parameters that is parsed with typesafe Config (e.g. from application.conf/model.conf)
 case class ConfigGlobalParameters(useLegacyMetrics: Option[Boolean],
                                   explicitUidInStatefulOperators: Option[Boolean],
-                                  useTypingResultTypeInformation: Option[Boolean])
+                                  useTypingResultTypeInformation: Option[Boolean],
+                                 //TODO: temporary, until we confirm which method works better
```

Comment:
1. How you will now that one of this method works better?
2. IMO Use I/O is confusing, because it can mean some other I/O. Maybe useIOMonad?

Reply:
1. I changed comment :)
2. :+1: